docs/usage.md

```diff
@@ -474,9 +476,15 @@ See the main [Nextflow documentation](https://www.nextflow.io/docs/latest/config
 
 If you have any questions or issues please send us a message on [Slack](https://nf-co.re/join/slack) on the [`#configs` channel](https://nfcore.slack.com/channels/configs).
 
-## Tutorials
+## Troubleshooting & FAQ
+
+### How to test the pipeline
+
+### Variantcalling tools
 
-### Which tool for which data type
+#### Which tool can be used for for which data type?
+
+This list is by no means exhaustive and it will depend on the specific analysis you would like to run. This is a suggestion based on the individual docs of the tools specifically for human genomes and a garden-variety sequencing run.
 
 | Tool            | WGS | WES |  Panel | Tumor |  Normal | Somatic |
 | :-------------- | :-: | :-: | :----: | :---: | :-----: | :-----: |
```

Comment:
Maybe we should consider converting all tools to lowercase and change mutect to mutect2